src/UI/src/components/editor/editor-component.js

```diff
@@ -52,4 +52,12 @@ export class EditorComponent {
         this.router.navigateToRoute('article', {slug: this.slug})
       })
   }
+
+  get canSave() {
+    if (this.type === 'post') {
```

Comment:
unnecessary if, both if and else are the same, so just make it the return line by itself.